docs/getting-started/my-account/README.md

```diff
@@ -161,24 +210,24 @@ You can also opt-in to receive **email alerts for other items (that are not work
 
 ## Set an avatar
 
-To change your profile picture in OpenProject you can set an avatar in your My account settings. Navigate to **Avatar** in the menu.
+To change your profile picture in OpenProject you can set an avatar in your **My account** settings. Navigate to **Avatar** in the menu.
+
+![openproject_my_account_avatar](openproject_my_account_avatar.png)
 
 OpenProject uses Gravatar as default profile image. It displays a preview of your avatar.
 
 Also, you can upload a **Custom Avatar** by choosing a Avatar to be uploaded from a file. Press the blue **Update** button to change your profile picture.
 
 > **Note**: The optimum size to upload a new profile picture is 128 by 128 pixel. Larger files will be cropped.
 
-![change your avatar](change-avatar.png)
-
 # Delete account
 
 You can delete your own account in **My account**.
 
-If you cannot see the entry **Delete account** in the **My account** side menu, make sure the option "Users allowed to delete their account" is [activated in the administration](../../system-admin-guide/users-permissions/settings/#user-deletion).
-
 To delete your account, select **Delete account** from the side menu and enter your login to confirm the deletion.
 
-![delete your account](delete-account.png)
+![openproject_my_account_delete_account](openproject_my_account_delete_account.png)
```

Comment:
@MayaBerd your email address is visible here. Should we fake some email?

Reply:
@marcalcobe I think it's fine. previously it showed Birthe's e-mail

Reply:
I think just one time the screenshot below is sufficient. Maybe show the expaneded menu here.

Reply:
changed the screenshot and used a user named "OpenProject User"